website/search/elastic_search.py

```diff
@@ -885,7 +889,8 @@ def create_index(index=None):
                     'programArea': NOT_ANALYZED_PROPERTY,
                     'provider': NOT_ANALYZED_PROPERTY,
                     'title': ENGLISH_ANALYZER_PROPERTY,
-                    'abstract': ENGLISH_ANALYZER_PROPERTY
+                    'schoolType': ENGLISH_ANALYZER_PROPERTY,
+                    'studyDesign': ENGLISH_ANALYZER_PROPERTY
```

Comment:
Guessing these should be `NOT_ANALYZED_PROPERTY`, similarly to the other optional metadata fields